platform/ios/docs/guides/Runtime Styling.md

```diff
@@ -0,0 +1,56 @@
+# Dynamically Styling Maps
+
+Mapbox's runtime styling features allow you direct control over every layer in your maps with code. It's now possible create dynamic maps and visualizations that aren't possible with other mobile mapping SDKs. 
+
+Runtime styling expands upon the design power of Mapbox Studio and exposes all of the same properties and attributes directly to mobile developers in our SDK.
+
+Beyond the custom styled maps that you can create with [Mapbox Studio](https://www.mapbox.com/mapbox-studio/), you can now change the look and feel of your map on the fly having maps in your app visually respond to user interaction or or context. Or leverage the power of OpenGL for highly performant and complex data visualizations. Now it's possible to mix in your own data and bring your map to life.
+
+## Example use cases
+
+As an example of what's possible with runtime styling, consider some of the following use cases:
+
+### Styling maps on the fly
+
+At runtime, you can tailor the map specifically to your user interface. Tweak colors, text, and icons to match the style to your brand.
+
+![dynamic styles](img/runtime-styling/DynamicStyles.gif "an example showing dynamic styles")
+
+For maps that aren't going to change in response to custom data or user interaction, consider creating a custom map style with [Mapbox Studio](https://www.mapbox.com/mapbox-studio/).
+
+### Map interactivity 
+
+You can customize the map to the point of having it respond dynamically based on the actions your users are taking. Increase the text size of streets while a user is driving, emphasize points of interest tailored a user's preferences, change your UI if user are near parks, trails, landmarks, or rivers. 
+
+![emojis](img/runtime-styling/Emoji.gif "an example showing emoji interaction")
+
+### Powerful data visualization
+
+Mapbox maps are built on top of OpenGL and can support rendering data without the traditional overhead of `UIView`-based map annotations. 
+
+Mapbox can support data visualizations that were slow or impossible with traditional map SDKs:
+
+![hex bins](img/runtime-styling/HexBins.gif "an example using hex bins")
+![population](img/runtime-styling/Population.gif "an example showing population density")
+![snow levels](img/runtime-styling/SnowLevels.gif "an example visualizing snow levels in the mountains")
+
+### Powerful Annotations
```

Comment:
Section headings in the API reference pages use title case, for consistency with Apple documentation. Most of the section headings here use sentence case, but “Powerful Annotations” is in title case. I’d be in favor of switching to sentence case across the board, but for now let’s be consistent within guides with either sentence or title case.